CONTRIBUTING.md

```diff
@@ -12,3 +12,22 @@ zealous and modify more lines than expected.
 
 The code must comply with [PEP8](https://www.python.org/dev/peps/pep-0008/).
 Running ``tox -e pep8`` will tell what is wrong.
+
+
+## Abbreviations
+
+When using abbreviations, try to be consistent. Here is a list of
+abbreviation commonly used in the existing code base.
+
+- acct: account
+- cid: container ID
```

Comment:
What about chunk id or content id ?

Reply:
We don't have common abbreviations for these ones. "Content" is almost the same as "object", so we could use `oid` for "object ID".